desidlas/prediction/test_prediction.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
```

Comment:
Please put this in `tests/`

Put it into a method, e.g. `def test_prediction()`
and decorate it with something like:

```
remote_data = pytest.mark.skipif(os.getenv('CNN_MODEL') is None,
                                 reason='test requires CNN model')
```

and put this before the method:

`@remote_data`

See this example:

https://github.com/FRBs/FRB/blob/main/frb/tests/test_frbsurveys.py

Reply:
Then you should put the model in a folder and have the 
enrvionmental variable `CNN_MODEL` hold it.  Something like that.

Let me know if you have any questions.

Reply:
Hi X, I have put the codes in test/ and add the 'remote_data' module as you suggested. I add a reminder to export the environmental variable in installing.rst  file. The automatic test seems successful now. Please check this PR. Thanks!